Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "mozjs_sys"
 description = "System crate for the Mozilla SpiderMonkey JavaScript engine."
 repository = "https://github.com/servo/mozjs/"
-version = "0.60.1"
+version = "1.60.0"
```

Comment:
`0.61`, it's nowhere near stable and then we would need to have version `2.60` etc. Let's not keep it in sync with SM upstream given that's not doable long-term (e.g. we already need a breaking bump now).